explore/Classifier_results-different_genes.py

```diff
@@ -0,0 +1,291 @@
+
+    
+      
+
+
+# In[9]:
+
+AUROCS_train_lung = AUROCS_train[0:len(genes_LungCancer)]
+AUROCS_test_lung = AUROCS_test[0:len(genes_LungCancer)]
+p_lung = p[0:len(genes_LungCancer)]
+
+AUROCS_train_ts = AUROCS_train[len(genes_LungCancer):len(genes_LungCancer)+len(genes_TumorSuppressors)]
+AUROCS_test_ts = AUROCS_test[len(genes_LungCancer):len(genes_LungCancer)+len(genes_TumorSuppressors)]
+p_ts = p[len(genes_LungCancer):len(genes_LungCancer)+len(genes_TumorSuppressors)]
+
+AUROCS_train_on = AUROCS_train[len(genes_LungCancer)+len(genes_TumorSuppressors):]
+AUROCS_test_on = AUROCS_test[len(genes_LungCancer)+len(genes_TumorSuppressors):]
+p_on = p[len(genes_LungCancer)+len(genes_TumorSuppressors):]
+
+def autolabel(rects,ax):
+    # attach some text labels
+    for rect in rects:
+        height = rect.get_height()
+        ax.text(rect.get_x() + rect.get_width()/2., 1.05*height,
+                '%.2f' % height,
+                ha='center', va='bottom')
+        
+
+width = 0.35       # the width of the bars
+
+
+# In[10]:
+
+# Plot for lung cancer
+
+ind_lung = np.arange(len(AUROCS_train_lung))  # the x locations for the groups
+
+fig_lung, ax_lung = plt.subplots(figsize=(15, 5))
+train_results_lung = ax_lung.bar(ind_lung, AUROCS_train_lung, width, color='g')
+
+test_results_lung = ax_lung.bar(ind_lung + width, AUROCS_test_lung, width, color='m')
+
+per_results_lung = ax_lung.bar(ind_lung, p_lung, 2*width, color='y')
+
+# add some text for labels, title and axes ticks
+ax_lung.set_ylabel('AUROCs')
+ax_lung.set_title('AUROCs for lung cancer genes')
+ax_lung.set_xlabel('List of genes')
+ax_lung.set_xticks(ind_lung + width)
+ax_lung.set_xticklabels(gene_symbols[0:len(genes_LungCancer)])
+ax_lung.set_yticks(np.arange(0, 1.11, 0.1))
+
+ax_lung.legend((train_results_lung[0], test_results_lung[0], per_results_lung[0]), ('Train', 'Test','%'))
+
+autolabel(train_results_lung,ax_lung)
+autolabel(test_results_lung,ax_lung)
+autolabel(per_results_lung,ax_lung)
+
+plt.show()
+
+
+# In[11]:
+
+# Plot for tumor suppressors
+
+ind_ts = np.arange(len(AUROCS_train_ts))  # the x locations for the groups
+
+fig_ts, ax_ts = plt.subplots(figsize=(15, 5))
+train_results_ts = ax_ts.bar(ind_ts, AUROCS_train_ts, width, color='g')
+
+test_results_ts = ax_ts.bar(ind_ts + width, AUROCS_test_ts, width, color='m')
+
+per_results_ts = ax_ts.bar(ind_ts, p_ts, 2*width, color='y')
+
+# add some text for labels, title and axes ticks
+ax_ts.set_ylabel('AUROCs')
+ax_ts.set_title('AUROCs for tumor suppressors genes')
+ax_ts.set_xlabel('List of genes')
+ax_ts.set_xticks(ind_ts + width)
+ax_ts.set_xticklabels(gene_symbols[len(genes_LungCancer):len(genes_LungCancer)+len(genes_TumorSuppressors)])
+ax_ts.set_yticks(np.arange(0, 1.11, 0.1))
+
+ax_ts.legend((train_results_ts[0], test_results_ts[0], per_results_ts[0]), ('Train', 'Test','%'))
+
+autolabel(train_results_ts,ax_ts)
+autolabel(test_results_ts,ax_ts)
+autolabel(per_results_ts,ax_ts)
+
+
+# In[12]:
+
+# Plot for oncogenes

```

Comment:
I suspect also that Cognoma will need to be smart about how CNVs are incorporated in the `Y` matrix. My thinking is that for oncogenes a copy number gain would define a 1 whereas for tumor suppressors a copy number loss would define a 1.